### PR TITLE
#164047076 add read time for an article

### PR DESCRIPTION
**Pull request by Kibetchirchir**
#### Description
___
When a user publishes an article he/she can see the approximate readtime  

#### Type of Change
___

- [x] New Feature

#### How has this been tested
___
- [x] Unit tests
 - [x]  End to end testing

#### Checklist
___
- [x] Tests are included to show/proof the feature works.

#### Pivotal Tracker
___
[164047076](https://www.pivotaltracker.com/story/show/164047076)

### CODE REVIEW

**Comment by JumaKahiga** [comment]
Great execution.

File: authors/apps/article/serializers.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 from authors.apps.authentication.serializers import UserSerializer
 from .models import Article, ArticleLikes, Rate, ArticleComment, ArticleFavourite
 from ..profiles.serializers import ProfileSerializer
+import readtime
```
Comment (kevpy) [inline authors/apps/article/serializers.py]: Let local imports come after 3rd party libraries. 
`import readtime` should come before any other local imports

**Review by JoanNgatia — CHANGES_REQUESTED** [review]
@Kibetchirchir Make the change on the order of imports and we'll be good to go on this PR

File: authors/apps/article/serializers.py (hunk 1 above)
Comment (Kibetchirchir) [reply authors/apps/article/serializers.py]: @kevpy thanks I have resolved as advised

File: authors/apps/article/serializers.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 from django.core.validators import MinValueValidator, MaxValueValidator
 from django.db.models import Avg
 from rest_framework import serializers
+import readtime
```
Comment (JoanNgatia) [inline authors/apps/article/serializers.py]: @Kibetchirchir  order of these imports should be readtime, django, rest_framework and then imports from local files.

**Review by JoanNgatia — CHANGES_REQUESTED** [review]

**Review by JoanNgatia — APPROVED** [review]

**Pull request merged**